pkg/dom0-ztools/Dockerfile

```diff
@@ -1,9 +1,57 @@
 ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:81e6520f4f1554789eb1ac299168e72ac37d88e2
 # hadolint ignore=DL3006
 FROM ${EVE_BUILDER_IMAGE} as zfs
-ENV PKGS zfs zfs-udev ca-certificates util-linux
+ENV BUILD_PKGS git patch ca-certificates util-linux build-base gettext-dev libtirpc-dev automake autoconf \
+    libtool linux-headers attr-dev e2fsprogs-dev glib-dev openssl-dev util-linux-dev
+ENV PKGS ca-certificates util-linux libintl libuuid libtirpc libblkid libcrypto1.1 zlib
```

Comment:
Why do we need to add packages (libintl libuuid libtirpc libblkid libcrypto1.1 zlib) due to building zfs and zfs-udev from source?

Reply:
When we install zfs package from Alpine we have the same subset of dependencies. We should install them explicitly now because we cannot rely on Alpine dependencies tree when building from source.